cookbooks/fb_users/README.md

```diff
@@ -9,9 +9,10 @@ Requirements
 Attributes
 ----------
 * node['fb_users']['user_defaults']
+* node['fb_users']['user_defaults']['gid']
+* node['fb_users']['user_defaults']['homedir_mode']
```

Comment:
I don't think this makes sense in `user_defaults`. If you want to change the _default_, then you should change `login.defs`. If you want to change for a _specific_user_, then you'd want it in this API under the specific `users` entry (as you have below).

Reply:
This brings a good point -- should we have an API in `fb_users` to manage `login.defs` ?

Reply:
I could argue it goes somewhere else, but I could also argue it goes here. If we wanna put it here I certainly have no objections.

Reply:
I'm going to address managing `/etc/login.defs` in a separate change